app/views/state_file/landing_page/edit.html.erb

```diff
@@ -65,7 +65,7 @@
               <% end %>
             <% else %>
               <div class="white-group spacing-below-60">
-                <%= t(".help_text_html") %>
+                <%= t(".#{@state_code}.help_text_html", default: ".help_text_html") %>
```

Comment:
[boulder]We still need to include the default text since this will now leave the other landing pages in a blank place for this section
![Screenshot 2024-10-28 at 12 47 13 PM](https://github.com/user-attachments/assets/f2de30a7-e05f-4872-92cf-572e670c29f3)

Reply:
@embarnard fixed the error, thanks for catching! Didn't realize default expects the actual text string